### PR TITLE
fix when no ATTR_COLOR is specified

### PR DESCRIPTION
**Pull request by Aohzan**
Hello,

When I call the service without specify a color, it raises an error as `lower()` doesn't exists because `call.data.get(ATTR_COLOR)` is not defined

### CODE REVIEW

**Comment by jcgoette** [comment]
I'll take a look at this tonight - can you go ahead and lint this with black?

File: custom_components/babybuddy/sensor.py [hunk 1]
```diff
@@ -107,7 +107,9 @@ def services_changes_add(call):
             ATTR_TIME: call.data.get(ATTR_TIME),
             ATTR_WET: call.data.get(ATTR_WET),
             ATTR_SOLID: call.data.get(ATTR_SOLID),
-            ATTR_COLOR: call.data.get(ATTR_COLOR).lower(),
+            ATTR_COLOR: call.data[ATTR_COLOR].lower()
+            if ATTR_COLOR in call.data
+            else None,
```
Comment (jcgoette) [inline custom_components/babybuddy/sensor.py:112]: Have you had a chance to test this? I don't think "color" will be in call.data.get(ATTR_SOLID). Thinking it should be more like this?

    `ATTR_COLOR: call.data.get(ATTR_COLOR).lower() if isinstance(call.data.get(ATTR_COLOR), str) else call.data.get(ATTR_COLOR)`

Comment (Aohzan) [reply custom_components/babybuddy/sensor.py:112]: Yes it works, ˋ.get()ˋ return ˋNone ˋ if the key doesn’t exist

**Comment by jcgoette** [comment]
Looks good, thank you @Aohzan!

**Pull request merged**